WordPress/Credentials/gencredentials.rb

```diff
@@ -177,9 +186,14 @@ def print_class(client, secret, sentry, appcenter, giphy, google_client, google_
 zendesk_app_id = nil
 zendesk_url = nil
 zendesk_client_id = nil
+encrytedlogkey=nil
+
 File.open(path) do |f|
   f.each_line do |l|
-    (k,value) = l.split("=")
+    eqPos = l.index("=")
+    k = l[0...eqPos]
+    value = l[(eqPos + 1)..-1]
```

Comment:
This change is required because otherwise values in the file that have a `=` in them are corrupted.